/**:
  ros__parameters:
    filter:
      near_clutter_range: 0.0
    map:
      frame_id: map
      length: 2500.0              # meters
      width: 2500.0               # meters
      resolution: 10.0            # meters/cell
      pub_interval: 0.1           # seconds
    max_queue_size: 1000
